wordpress/wp-content/plugins/sil-dictionary-webonary/include/searchform_func.php

```diff
@@ -279,25 +280,7 @@ function theCursorPosition(ofThisInput) {
 					</select>
 					<?php echo $parts_of_speech_dropdown; ?>
 					<br>
-					<?php
-					$checkedWholeWords = "";
-
-					if(isset($_GET['search']))
-					{
-						if(isset($_GET['match_whole_words']))
-						{
-							$checkedWholeWords = "checked";
-						}
-					}
-					else
-					{
-						if(get_option('include_partial_words') == 0)
-						{
-							$checkedWholeWords = "checked";
-						}
-					}
-					?>
-					<input name="match_whole_words" value="1" <?php echo $checkedWholeWords; ?> type="checkbox"> <?php _e('Match whole words', 'sil_dictionary'); ?>
+                    <input id="match_whole_words" name="match_whole_words" value="1" checked type="checkbox"> <label for="match_whole_words"><?php _e('Match whole words', 'sil_dictionary'); ?></label>
```

Comment:
Perhaps we need to confirm with Product Owner, but I think if the user has deselected this checkbox, then it should not be selected. So we need to check the value of query parm `$_GET['match_whole_words']` and set the checked value accordingly, as in `include_partial_words`

Reply:
Also, I think we also need to check with Product Owner and probably remove the `always include searching through partial words` in Webonary plugin Search config, since this becomes moot with this fix. 
![Screen Shot 2020-09-15 at 8 57 29 PM](https://user-images.githubusercontent.com/60716623/93283778-d5a90380-f796-11ea-969d-f8c0e1c6765b.png)